andino_apps/package.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>andino_apps</name>
+  <version>0.1.0</version>
+  <description>Package for apps created with andino</description>
+  <author email="jesus.silva@ekumenlabs.com">JesusSilvaUtrera</author>
+  <maintainer email="jesus.silva@ekumenlabs.com">JesusSilvaUtrera</maintainer>
+  <maintainer email="franco.c@ekumenlabs.com">Franco Cipollone</maintainer>
+  <license file="LICENSE">BSD Clause 3</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <exec_depend>ros2launch</exec_depend>
+  <exec_depend>andino_gz_classic</exec_depend>
+  <exec_depend>nav2_bringup</exec_depend>
+  <exec_depend>turtlebot3_gazebo</exec_depend>
```

Comment:
Do we need to be explicit about turtlebot3_gazebo. IIRC nav2_bringup manages that dependency.
We are not directly using any turtlebot_gazebo asset. (If we use the world we use it via nav2_bringup)